canbench-bin/src/fmt.rs

```diff
@@ -1,5 +1,5 @@
-const POSITIVE_INFINITY_PERCENT: &str = "1.0E100%";
-const NEGATIVE_INFINITY_PERCENT: &str = "-1.0E100%";
+const POSITIVE_INFINITY_PERCENT: &str = "1.0E99";
```

Comment:
Shouldn't we always have the percent sign at the end?

Reply:
google sheets removes % for scientific format any ways which turns 1e100% into 1e98.
I suppose it's better to have 1e99 so you can easily guess it's some kind of a maximum number.